src/main/java/uk/gov/ons/ctp/response/sample/domain/model/SampleSummary.java

```diff
@@ -66,4 +68,16 @@
   @Column(name = "expectedcollectioninstruments")
   private Integer expectedCollectionInstruments;
 
+  @JsonIgnore
+  public Integer getSampleSummaryPK() {
+    return sampleSummaryPK;
+  }
+
+  @Column(name = "notes")
+  @Size(max=255)
+  private String notes;
+
+  @Column(name = "description")
+  @Size(max=250)
```

Comment:
It's a little odd that description is 250 and notes 255 - feels like they should be the same

Reply:
I don't know why description is 250 (might be a business requirement) so can't change that - I can change notes to 250 but is this really necessary?

Reply:
Is it necessary? Nope. Is it worth doing? Probably, may as well make our DB schema consistent now rather than looking at it in a few months time and going "why the heck is one 250 and the other 255?"